src/v8/prettier-dev-dependencies.ts

```diff
@@ -0,0 +1,27 @@
+import { existsSync } from "fs";
+
+import { sh } from "../util/sh";
+
+/**
+ * Updates prettier to v3 and applies prettier changes to all source files
+ */
+export default async function updatePrettierAndUpdateSourceStyles() {
+    if (existsSync("package.json")) {
```

Comment:
is comet v8 already installed when running this script?

Reply:
Yes, but will likely fail due to misconfigured peer dependencies. Should be changed to something like this: https://github.com/vivid-planet/comet-upgrade/blob/main/src/v8/update-mikro-orm-dependencies.ts. Note the extra `stage` export.